app/components/content_list_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ContentListComponent < ViewComponent::Base
+  def initialize(heading:, list:)
+    @heading = heading
+    @list = list
+  end
+
+  def href_anchor(section_heading)
+    "##{section_heading.gsub(' ', '-')}"
```

Comment:
```suggestion
    "##{section_heading.parameterize}"
```

I'd use `parameterize` otherwise you'll have funky punctuation in the anchor references